README.md

```diff
@@ -29,6 +29,20 @@ Note that all flags can be replaced with environment variables; for instance,
 `--dry-run` could be replaced with `EXTERNAL_DNS_DRY_RUN=1`, or
 `--registry txt` could be replaced with `EXTERNAL_DNS_REGISTRY=txt`.
 
+## Deploying to a Cluster
+
+The following tutorials are provided:
+
+* [AWS](docs/tutorials/aws.md)
+* [Azure](docs/tutorials/azure.md)
+* [Cloudflare](docs/tutorials/cloudflare.md)
+* [DigitalOcean](docs/tutorials/digitalocean.md)
+* Google Container Engine
+	* [Using Google's Default Ingress Controller](docs/tutorials/gke.md)
+	* [Using the Nginx Ingress Controller](docs/tutorials/nginx-ingress.md)
+
+## Running Locally
```

Comment:
What are you thinking that we should fill it with?

I added the links because when I found the project, I wasn't interested in running it locally but instead to deploy it to my cluster and I couldn't find any info on how actually deploy it to a cluster. Google pointed me to the excellent tutorial section, so I thought I'd add links to those in the `README.md` as long term I think many more people will fall into the category of just deploying it.